### PR TITLE
Add: Pie chart subplot title

### LINKED ISSUE #2035: Pie Chart Title
Hi there,

there does not seem to be any way to add a title to the pie chart when one is displaying multiple pie charts per visualization. Could the title (which is shown on hover) be added above each of the charts?

### PR DESCRIPTION
**Pull request by deecay**
pie-subplot-title

Closes #2035 and #1180.

### CODE REVIEW

**Comment by arikfr** [comment]
Nice!

File: client/app/visualizations/chart/plotly.js [hunk 1]
```diff
@@ -252,7 +252,9 @@ const PlotlyChart = () => {
           const cellWidth = 1 / cellsInRow;
           const cellHeight = 1 / rows;
           const xPadding = 0.02;
-          const yPadding = 0.05;
+          const yPadding = 0.1;
```
Comment (arikfr) [inline client/app/visualizations/chart/plotly.js]: Why did we need to change this?

Comment (deecay) [reply client/app/visualizations/chart/plotly.js]: We need a little more space than now to make the second-row title to fit. The gap between the first row and second row is too tight for a row of text to fit nicely.

**Comment by arikfr** [comment]
Thanks!

**Comment by deecay** [comment]
FYI, Upcoming Plotly will have native support for Pie subplot titles.
[link](https://github.com/codrut3/plotly.js/blob/bce5add399f962241b6fd482f1eefa19b0d601d7/test/image/baselines/pie_title_variations.png)

**Comment by arikfr** [comment]
Nice :) Thanks for the update, @deecay .

**Pull request merged**